spec/workers/company_domain_worker_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+RSpec.describe CompanyDomainWorker, sidekiq: true do
+  let!(:company) { create(:company, :worker_domain) }
+  let!(:worker) { CompanyDomainWorker.new }
+
+  before(:each) do
+    allow(worker).to receive(:link_open).and_return(response_html)
+
+    worker.perform(company.id)
+  end
+
+  describe 'with valid data' do
+    let!(:response_html) { Nokogiri::HTML(open('spec/factories/test.html')) }
+
+    it 'testing worker queueing' do
+      expect { CompanyDomainWorker.perform_async(1) }.to change(CompanyDomainWorker.jobs, :size).by(1)
+    end
+
+    it 'worker company should be as created one' do
+      expect(worker.company).to eq(company)
+    end
+
+    it 'sub_links content should be correct. not nil, empty, outside' do
```

Comment:
could you make description more morrect?

> sub_links content should be correct. not nil, empty, outside

so, 
* should be correct
* not nil
* empty  (not nil but empty?)
* outside